web/helpers/product-creator.js

```diff
@@ -83,17 +83,27 @@ export default async function productCreator(session, count = DEFAULT_PRODUCTS_C
   const client = new Shopify.Clients.Graphql(session.shop, session.accessToken);
 
   for (let i = 0; i < count; i++) {
-    await client.query({
-      data: {
-        query: CREATE_PRODUCTS_MUTATION,
-        variables: {
-          input: {
-            title: `${randomTitle()}`,
-            variants: [{ price: randomPrice() }],
+    try {
+      await client.query({
+        data: {
+          query: CREATE_PRODUCTS_MUTATION,
+          variables: {
+            input: {
+              title: `${randomTitle()}`,
+              variants: [{ price: randomPrice() }],
+            },
           },
         },
```

Comment:
We also need to capture the `errors` component of the body on a 200 response.
```suggestion
    try {
      const response = await client.query({
        data: {
          query: CREATE_PRODUCTS_MUTATION,
          variables: {
            input: {
              title: `${randomTitle()}`,
              variants: [{ price: randomPrice() }],
            },
          },
        },
      });
      if (response.body.errors not empty) {
        // bork
      }
```